hack/verify-golint.sh

```diff
@@ -18,34 +18,5 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-# shellcheck source=/dev/null
```

Comment:
You should be able to remove this whole file, since the invocation of `verify-golangci-lint` should already invoke the linters we are requesting here.